engine/src/lib.rs

```diff
@@ -53,12 +63,39 @@ impl ServalEngine {
             .build();
 
         let mut store = Store::new(&self.engine, wasi);
+
         let module = Module::from_binary(&self.engine, binary)?;
 
+        // Load any custom WASM node features that the job requires (...and that we have)
+        println!("Module wants these imports:");
+        let required_modules = module
+            .imports()
+            .map(|import| import.module().to_string())
+            .filter(|import| !import.starts_with("wasi_snapshot_"));
```

Comment:
Everything that uses WASI is going to try to import `wasi_snapshot_preview_1`; that's provided by `wasmtime_wasi` for us.